From 07ec2ad34c42dba8656d3f543164f360f481c52e Mon Sep 17 00:00:00 2001
From: Daniel Dadap <ddadap@nvidia.com>
Date: Thu, 15 May 2025 08:32:35 -0500
Subject: [PATCH] conf: aliases: add hda-acpi -> HDA-Intel alias

The new snd_hda_acpi driver in Linux exposes the existing Azalia
interface to non-PCI devices advertised over ACPI. Add an alias
to the existing HDA-Intel configuration file so that devices using
this driver can be discovered properly.

Signed-off-by: Daniel Dadap <ddadap@nvidia.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>
---
 src/conf/cards/aliases.conf | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/conf/cards/aliases.conf b/src/conf/cards/aliases.conf
index a54824ae..e2d59aa7 100644
--- a/src/conf/cards/aliases.conf
+++ b/src/conf/cards/aliases.conf
@@ -57,6 +57,7 @@ CMI8786 cards.CMI8788
 CMI8787 cards.CMI8788
 pistachio cards.pistachio-card
 VC4-HDMI cards.vc4-hdmi
+hda-acpi cards.HDA-Intel
 
 <confdir:ctl/default.conf>
 <confdir:pcm/default.conf>
-- 
2.49.0

